### ISSUE TITLE
improved nsfw markings

### ISSUE THREAD

**Issue by bobkinect98** [powercord-community/suggestions#267]
### Description
it marks channel/users/groups/servers/dms nsfw but client sided
it also marks server sided nsfw channels with a nsfw mark next to their name
### More info

----

 - [x] I checked if this request wasn't already proposed
 - [x] I made sure this request follows Powercord's [plugin guidelines](https://powercord.dev/guidelines)

**Comment by MulverineX** [comment]
Ideally, this would work with [BlurNSFW](https://github.com/cloudrac3r/powercord-blurnsfw) as well.

**Comment by bobkinect98** [comment]
thats not what i meant, but i barely stated any info on what i wanted, so ill edit the post with more info

**Comment by MulverineX** [comment]
@bobkinect98 nope, you were clear, I just meant that channels/user/group DM's that you flag as NSFW with this plugin would ideally also be detected by that plugin.
